spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/ReorderMemory.scala

```diff
@@ -85,17 +90,45 @@ class ReorderMemory(inputFormat: MemoryData, outputFormat: MemoryData,
       }
     }
 
-    val fwdReorderPrimDesc = MklDnn.ReorderPrimitiveDescCreate(
-      realInput(0).getPrimitiveDescription(runtime),
-      realOutput(0).getPrimitiveDescription(runtime))
+    val needNotAttr = inputFormats()(0).dataType == DataType.F32 &&
```

Comment:
you can add more comments for those operations

Reply:
OK.